integration/basic_test.go

```diff
@@ -447,6 +447,43 @@ func TestDatabaseName(t *testing.T) {
 
 	t.Parallel()
 
+	t.Run("SpecialCharacters", func(t *testing.T) {
+		ctx, collection := setup.Setup(t)
+		for name, tc := range map[string]struct {
+			db string // database name, defaults to empty string
+
+			err        *mongo.CommandError // required, expected error from MongoDB
+			altMessage string              // optional, alternative error message for FerretDB, ignored if empty
+			skip       string              // optional, skip test with a specified reason
+		}{
+			"Dash": {
+				db: "--",
+			},
+			"Underscore": {
+				db: "__",
+			},
+			"Sqlite": {
+				db: "sqlite_",
+			},
```

Comment:
Could you merge `63ok` test case around L587 to this table test please :pray:

Reply:
That was done, resolving